### ISSUE TITLE
Unable to set breakpoints in non-Bazel projects

### ISSUE THREAD

**Issue by heemskerkerik** [bazelbuild/intellij#5673]
### Description of the bug:

As the title says, I’ve found out that the Bazel plugin was preventing me from setting breakpoints in non-Bazel projects. Disabling the plugin and restarting immediately fixed the problem, and after re-enabling the plugin and restarting, the problem immediately resurfaced.

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

- Create a new Maven-bazed project (File, New Project)
  - Select Java as the language, Maven as the Build System
  - Make sure ‘Add sample code’ is checked
- Go to the sample code and try to set a breakpoint on any line of code, either by clicking the line number in the gutter or by putting the caret on a line and invoking the ‘Toggle line breakpoint’ action.
- Nothing happens.

### Which Intellij IDE are you using? Please provide the specific version.

 IntelliJ IDEA 2023.2.5

### What programming languages and tools are you using? Please provide specific versions.

Java (azul-sdk 18.0), Maven

### What Bazel plugin version are you using?

2023.10.24.0.1-api-version-232

### Have you found anything relevant by searching the web?

Nope.

### Any other information, logs, or outputs that you want to share?

```java.lang.IllegalStateException: Non-blaze project is provided
	at com.google.common.base.Preconditions.checkState(Preconditions.java:502)
	at com.google.idea.blaze.base.sync.data.DelegatingBlazeProjectDataManager.lambda$new$0(DelegatingBlazeProjectDataManager.java:39)
	at com.google.common.base.Suppliers$NonSerializableMemoizingSupplier.get(Suppliers.java:183)
	at com.google.idea.blaze.base.sync.data.DelegatingBlazeProjectDataManager.getBlazeProjectData(DelegatingBlazeProjectDataManager.java:50)
	at com.google.idea.blaze.skylark.debugger.SkylarkDebuggingUtils.debuggingEnabled(SkylarkDebuggingUtils.java:44)
	at com.google.idea.blaze.skylark.debugger.impl.SkylarkLineBreakpointType.canPutAt(SkylarkLineBreakpointType.java:43)
...
```

**Comment by mai93** [comment]
@ujohnny is this fixed with https://github.com/bazelbuild/intellij/commit/59ddcbe54739726f32e2953002d8a49d8f91c5cf?

**Comment by ujohnny** [comment]
> @ujohnny is this fixed with [59ddcbe](https://github.com/bazelbuild/intellij/commit/59ddcbe54739726f32e2953002d8a49d8f91c5cf)?

yes, it is.

duplicates #5655
